Readme_ES.md

```diff
@@ -0,0 +1,307 @@
+# Paquete GT para moderadores de New Horizons
+
+## Paquete para moderadores verificado a través de un ajuste a los moderadores (LOS MODERADORES OBTIENEN AUTORIZACIÓN)
+
+
+Versión 2.0.2.7 is out 15.01.2017
+
+[![estado de compilación](http://jenkins.usrv.de:8081/buildStatus/icon?job=GTNewHorizons%20Configs)](http://jenkins.usrv.de:8081/job/GTNewHorizons%20Configs/)
+
+## Un paquete para moderadores más grande con cada especificación verificada
```

Comment:
But there's more: "A package for moderators bigger with every verified specification".